### PR TITLE
Fix nav2_velocity_smoother for sim time

### PR DESCRIPTION
**Pull request by HovorunB**
---

## Basic Info

| Info | Fix nav2_velocity_smoother for sim time |
| ------ | ----------- |
| Ticket(s) this addresses   |  |
| Primary OS tested on | Ubuntu  |
| Robotic platform tested on | Gazebo simulation |

---

## Description of contribution in a few bullet points

If sim time is used: `now()` might be close to 0 on startup and `last_command_time_` is defined 0 making the condition `now() - last_command_time_ > velocity_timeout_` False, which assumes that velocity command was received recently and then leads to an error when trying to access command_'s properties ( `command_->linear.x` )
So i added check for command_ being null before proceeding with smoothing

## Description of documentation updates required from your changes

---

#### For Maintainers: 
- [ ] Check that any new parameters added are updated in navigation.ros.org
- [ ] Check that any significant change is added to the migration guide
- [ ] Check that any new features **OR** changes to existing behaviors are reflected in the tuning guide
- [ ] Check that any new functions have Doxygen added
- [ ] Check that any new features have test coverage
- [ ] Check that any new plugins is added to the plugins page
- [ ] If BT Node, Additionally: add to BT's XML index of nodes for groot, BT package's readme table, and BT library lists

### CODE REVIEW

File: nav2_velocity_smoother/src/velocity_smoother.cpp [hunk 1]
```diff
@@ -209,6 +209,11 @@ double VelocitySmoother::applyConstraints(
 
 void VelocitySmoother::smootherTimer()
 {
+  // Wait until the first command is received
+  if (command_==nullptr) {
```
Comment (SteveMacenski) [inline nav2_velocity_smoother/src/velocity_smoother.cpp:213]:
```suggestion
  if (!command_) {
```

Comment (SteveMacenski) [reply nav2_velocity_smoother/src/velocity_smoother.cpp:213]: Fixes both a linting issue and a style issue

Comment (HovorunB) [reply nav2_velocity_smoother/src/velocity_smoother.cpp]: Thank you!

**Review by SteveMacenski — APPROVED** [review]

**Review by AlexeyMerzlyakov — APPROVED** [review]
The change LGTM: the main logic is comes from `smootherTimer()` callback, so we need to wait until first message to be received.

**Pull request merged**